platform-hub-api/spec/controllers/me_controller_spec.rb

```diff
@@ -42,6 +42,51 @@
     end
   end
 
+  describe '#delete_identity' do
+    it_behaves_like 'authenticated' do
+      before do
+        create(:identity,
+          user: current_user,
+          external_id: '1234567',
```

Comment:
For the purpose of this test, you probably don't need to explicitly set this field and all the ones below it.